javascript/builtins/Iterator.json

```diff
@@ -42,6 +42,553 @@
             "deprecated": false
           }
         },
+        "Iterator": {
+          "__compat": {
+            "description": "<code>Iterator()</code> constructor"
```

Comment:
Line looks fine, except that you missed the trailing comma, which is causing the linter test to fail. 

(I would've just fixed it and then merged, but it isn't letting me edit the file for some reason; I think maybe I don't have the right permissions)

Reply:
```suggestion
            "description": "<code>Iterator()</code> constructor",
```